apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -386,10 +388,36 @@ Contacts.prototype = {
     }
   },
 
+  checkContactCreated: function() {
+    var ci = this.client.findElement(Contacts.Selectors.contactsIframe);
+    this.client.switchToFrame(ci);
+
+    var contactInContactList = this.client.findElement
+      (Contacts.Selectors.listContactFirst);
+    var contactName = contactInContactList.text();
+    return contactName;
+  },
+
   tapSave: function() {
     this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
   },
 
+  tapSavefromDialer: function() {
+    this.client.helper.waitForElement(Contacts.Selectors.formGivenName);
+    this.client.helper.fillInputField(Contacts.Selectors.formGivenName, 'Name');
+    this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
+    
+    var DialerOptionsView = require
```

Comment:
Manually, after a new contact is created you return on the keypad, which is also the first view open on the dialer. Then I'd just return `Dialer`. Returning the view of the tabs seems to restrictive/